crmsh/corosync.py

```diff
@@ -386,6 +386,15 @@ def add_node_ucast(IParray, node_name=None):
     f = open(conf()).read()
     p = Parser(f)
 
+    # to check if the same IP already configured
+    exist_iplist = []
+    for path in p.all_paths():
+        if re.search('nodelist.node.ring[0-9]*_addr', path):
+            exist_iplist += p.get_all(path)
```

Comment:
I would use `append` instead of `+=`, it's a bit more visual

Reply:
get_all will return a list
and here I want to extend the list `exist_iplist`, not append

Reply:
You could use `exist_iplist.extend(p.get_all(path))`, but I am fine with `+=`